tsdb/db_test.go

```diff
@@ -426,64 +426,65 @@ func TestDeleteSimple(t *testing.T) {
 		},
 	}
 
-Outer:
 	for _, c := range cases {
-		db := openTestDB(t, nil, nil)
-		defer func() {
-			require.NoError(t, db.Close())
-		}()
+		t.Run("", func(t *testing.T) {
```

Comment:
Could we introduce test case names, to easily tell them apart if they should break?

Reply:
Numbers seem fine to me.  There's only 5.